docs/deployment/registries.md

```diff
@@ -0,0 +1,116 @@
+docker.pkg.github.com/yourusername/rdmrepo/rdmimage:latest      latest              9b6dd5ae6b48        17 hours ago        2.33GB
+```
+
+The last step is to push your package to the github registry:
+
+``` console
+docker push docker.pkg.github.com/yourusername/rdmrepo/rdmimage:latest
+```
+
+In order to use this image, you must set the following value in your `values.yaml` file (for both `web` and `worker`):
+
+```
+image: docker.pkg.github.com/yourusername/rdmrepo/rdmimage:latest 
+```
+
+Even if your project and/or image is public, GitHub requires you to be authenticated in order to pull your image.
+In OpenShift you can change the default pulling configuration so that it uses your token. You can do so as follows:
+
+``` console
+$ oc create secret docker-registry <SECRET_NAME> \
+    --docker-server=docker.pkg.github.com \
+    --docker-username=<YOUR_GITHUB_USERNAME> \
+    --docker-password=<YOUR_GITHUB_PASSWORD>
+$ oc secrets link default <SECRET_NAME> --for=pull \
+    --namespace=<YOUR_OPENSHIFT_PROJECT_NAME>
+```
+
+## OpenShift
+
+First you need to login in your OpenShift cluster and its image registry:
+
+``` console
+$ docker login -u openshift -p $(oc whoami -t) <registry_ip>:<port>
+$ oc login
+$ oc project
+Using project "inveniordm" on server "<registry_ip>:<port>".
+```
+
+Then find your docker image and tag it you the url of your OpenShift registry:
+
+``` console
+$ docker images
+REPOSITORY                                          TAG                 IMAGE ID            CREATED             SIZE
+demo-inveniordm                                     latest              9b6dd5ae6b48        17 hours ago        2.33GB
+
+$ docker tag demo-inveniordm <registry_ip>:<port>/inveniordm/demo-inveniordm:latest
+```
+
+!!! info "Project name"
+    Note that `inveniordm` is the name of the OpenShift project we are using. It's value should be the one returned
+    by the `oc project` command. Therefore the tag is: ``<registry_ip>:<port>/<project_name>/<name_of_your_image>:<version>``
+
+Check that it was tagged correctly:
+
+``` console
+$ docker images                                                                   
+REPOSITORY                                          TAG                 IMAGE ID            CREATED             SIZE
+demo-inveniordm                                     latest              9b6dd5ae6b48        17 hours ago        2.33GB
+<registry_ip>:<port>/inveniordm/demo-inveniordm     latest              9b6dd5ae6b48        17 hours ago        2.33GB
+```
+
+Finally push it to the image registry of OpenShift:
+
+``` console
+$ docker push <registry_ip>:<port>/inveniordm/demo-inveniordm:latest 
+```
+
+In order to use this image, you must set the following value in your `values.yaml` file (for both `web` and `worker`):
+
+``` console
+image: <registry_ip>:<port>/inveniordm/demo-inveniordm:rdm 
```

Comment:
is this different helm variable than the `web`/`worker`? Because then we have 3 variables that they expect the pushed image...

Reply:
Nope, it is the same. I change it to reflect that is the part under `web` and `worker`.